src/jsonLogic.js

```diff
@@ -107,32 +109,106 @@ export function yupSchemaWithCustomJSONLogic({ field, validations, config, id })
     );
 }
 
+function replaceHandlebarsTemplates({
+  value: toReplace,
+  validations,
+  formValues,
+  parentID,
+  name: fieldName,
+}) {
+  if (typeof toReplace === 'string') {
+    return toReplace.replace(/\{\{([^{}]+)\}\}/g, (match, key) => {
+      return validations
+        .getScope(parentID)
+        .evaluateComputedValueRuleForField(key.trim(), formValues, fieldName);
+    });
+  } else if (typeof toReplace === 'object') {
```

Comment:
Do you have a case where the `toReplace` is an object? Looking at the `schemaWithComputedAttributesAndErrorMessages` even `x-jsf-errorMessage` doesn't enter here.

Reply:
Same with tests, none of the tests cover this `else` branch

Reply:
I was very confused for _why_ this wasn't being called but it was a mistake on my part. This actually goes with the inline rules code. I'll shift it to the appropriate PR. It will be removed for this PR.